doc/first-steps.ipynb

```diff
@@ -205,6 +207,24 @@
     "fig.savefig('first-steps-central-america.png')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Also you can call `fig.savefig` to save the figure to a file and then open the figure in an external viewer:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "fig.savefig('first-steps-central-america.png', show=True)"
```

Comment:
This might be a bit problematic as a code cell. The notebook is executed every time we build the documentation and it would be bad for a window to open whenever that happens. 

Could you please include this as a Markdown code block inside the previous cell? Like I did for `fig.show(external=True)`.